apps/src/gamelab/GameLabSprite.js

```diff
@@ -86,17 +86,7 @@ module.exports.createSprite = function (x, y, width, height) {
    * @private
    * For game lab, don't update the animation sizes because all frames are the same size.
    */
-  s._syncAnimationSizes = function (animations, currentAnimation) {
-    //has an animation but the collider is still default
-    //the animation wasn't loaded. if the animation is not a 1x1 image
-    //it means it just finished loading
-    if (this.colliderType === 'default' && animations[currentAnimation].getWidth() !== 1 &&
-      animations[currentAnimation].getHeight() !== 1) {
-      this.collider = this.getBoundingBox();
-      this.colliderType = 'image';
-      //quadTree.insert(this);
-    }
-  };
+  s._syncAnimationSizes = function (animations, currentAnimation) {};
```

Comment:
Yes. The [original method](https://github.com/molleindustria/p5.play/blob/master/lib/p5.play.js#L1193) does two things: It updates the sprite to match the animation dimensions, and does the same with the collider.

Back in July, @caleybrock [extracted `_syncAnimationSizes` and modified our version](https://github.com/code-dot-org/code-dot-org/commit/ad499a1) to not update the sprite size with animation frame changes (since in gamelab frame sizes are uniform within an animation).

With the new collision system, [we stopped manually updating collider sizes](https://github.com/code-dot-org/p5.play/blob/master/lib/p5.play.js#L1248) - we delegate that to the colliders themselves.

So, this helper collapses into a no-op in our custom version.